### ISSUE TITLE
interdocument xref not resolving correctly

### ISSUE THREAD

**Issue by rosieyohannan** [asciidoctor/asciidoctor-pdf#1146]
I'm running: Asciidoctor PDF 1.5.0.alpha.18 using Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin18]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)

I'm trying to single source Jekyll and PDF output from Asciidoc and have it mostly working now except interdocument xrefs.

I have a wrapper `.adoc` for the PDF and am using `:include:` to put the various chapters into one big PDF, and these individual chapters are making up pages of the jekyll site.

Here's how the 'wrapper' for the PDF is looking:
<img width="452" alt="Screenshot 2019-07-09 at 12 16 36" src="https://user-images.githubusercontent.com/24589403/60883928-715dec00-a243-11e9-8647-bf1842f86b85.png">

I'm using `:relfileprefix:` here as I'm using annother `:relfileprefix`: set form the Jekyll `_config.yml` to resolve inter links in the jekyll site, an this appears to be working correctly. 

I am trying to get one link working between Overview:

<img width="1419" alt="Screenshot 2019-07-09 at 12 19 14" src="https://user-images.githubusercontent.com/24589403/60884045-d3b6ec80-a243-11e9-9629-593538fa65fd.png">

and the other file `nomad.adoc`:
<img width="823" alt="Screenshot 2019-07-09 at 12 20 22" src="https://user-images.githubusercontent.com/24589403/60884084-f3e6ab80-a243-11e9-84e2-036aa35abf96.png">

But when I generate the PDF, and try clicking on the xref link, its looking for `nomad.pdf` plus the section ID .... I'm really not sure where to go from here 🤔

<img width="717" alt="Screenshot 2019-07-09 at 12 22 03" src="https://user-images.githubusercontent.com/24589403/60884175-301a0c00-a244-11e9-9570-6091b2b9fd74.png">

Any help would be VERY much appreciated as I'm so close to having this working :-)

**Comment by rosieyohannan** [comment]
Repo here if helpful: https://github.com/rosieyohannan/circleci-docs/tree/new/asciidoc

**Comment by mojavelinux** [comment]
Very helpful. That will allow me to investigate. Thanks!

**Comment by rosieyohannan** [comment]
Hey @mojavelinux, just wondering if you can confirm if interdocument cross referencing _should_ work in the situation I've described here (single source to PDF and jekyll)? I've been working on a proof of concept for moving over some public docs to Asciidoc+pdf+jekyll from markdown+pandoc+jekyll  but this issue is kind of a make or break - I can work round it not working now as long as the fact its not working is a bug (or something I'm doing wrong) rather than a new feature or impossibility. Thanks! :-)

**Comment by rosieyohannan** [comment]
I worked out what I was doing wrong, finally! I hadn't specified the full path location to the included file in the xref. Same kind of thing as https://github.com/asciidoctor/asciidoctor-pdf/issues/809. Can close this now! Maybe I can submit a PR to the docs to add what worked @mojavelinux ? :-D

**Comment by mojavelinux** [comment]
That's absolutely fantastic to hear.

Yes, I would love to receive a PR with docs to explain what worked. (Don't fret too much about where it belongs. What's most important is that we capture the info). Thanks!!

**Comment by rosieyohannan** [comment]
I'm thinking about what to add here and am wondering if it's actually the desired behaviour that anchors *aren't* automatically generated for top level (=) section headings? I have :sectanchors: enabled... I'm re-looking at this because using [.anchors] above the top level headings is now giving me some errors when I try and deploy, although only in the html-proofer tests we have set up, the actual cross references do appear to be working.

Also the specifics of how I got this to work would probably only be of interest to people wanting Jekyll (or another static site) and PDF from a single source, I guess that's still useful to have in?
